src/components/scene/canvas.js

```diff
@@ -29,6 +29,8 @@ module.exports.Component = register('canvas', {
       canvas.style.height = data.height + '%';
       canvas.style.width = data.width + '%';
       scene.appendChild(canvas);
+    } else {
+      canvas.provided = true;
```

Comment:
personally, I would instead set a property on the `a-canvas` instead of the user-defined `canvas`